projects.json

```diff
@@ -4620,5 +4620,39 @@
         "action": "TestSwiftPackage"
       }
     ]
+  },
+  {
+    "repository": "Git",
+    "url": "https://github.com/apple/swift-testing",
```

Comment:
cc @ahoppen will adding this project cause issues for the stress tester?

Reply:
Since you don’t put the `source kit-smoke` tag on there, I think it should be fine. Thanks for checking.